app/models/bitflyer_connection.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+class BitflyerConnection < ApplicationRecord
+  SUPPORTED_CURRENCIES = ["BAT", "USD", "BTC", "ETH"].freeze
+
+  belongs_to :publisher
+
+  validates :recipient_id, uniqueness: true, allow_blank: true
```

Comment:
Would you be able to group these together based on the order of operations? Also, I don't think we need so much whitespace lol

Reply:
For sure, fixed